template/admin/initServer.sqf

```diff
@@ -6,11 +6,16 @@ publicVariable QMGVAR(admins);
 MGVAR(debug_admins) = [DEBUG_ADMINS];
 publicVariable QMGVAR(debug_admins);
 
-// Mathematical Heresy
+// Calculate time until start (0 if debug mode enabled)
+#ifdef DEBUG_MODE_FULL
+GVAR(timeUntilStart) = 0;
+#else
 systemTimeUTC params ["", "", "", "_hour", "_minute"];
 private _startTime = 14; // 1400z always
 GVAR(timeUntilStart) = ((_startTime * 60) - (_hour * 60 + _minute)) * 60; // start time - current time = time until start time
 GVAR(timeUntilStart) = GVAR(timeUntilStart) max 0;
+#endif
```

Comment:
```suggestion

systemTimeUTC params ["", "", "", "_hour", "_minute"];
private _startTime = 14; // 1400z always
GVAR(timeUntilStart) = ((_startTime * 60) - (_hour * 60 + _minute)) * 60; // start time - current time = time until start time
GVAR(timeUntilStart) = GVAR(timeUntilStart) max 0;

#ifdef DEBUG_MODE_FULL
GVAR(timeUntilStart) = 0;
#endif
```
This would be cleaner and you could debug calculations even with debug mode.